models/base_model.go

```diff
@@ -0,0 +1,77 @@
+package models
+
+import (
+	"errors"
+
+	"github.com/sirupsen/logrus"
+	"github.com/transcovo/go-chpr-logger"
+	mgo "gopkg.in/mgo.v2"
+)
+
+/*
+ErrGenericDB is the error returned when an unknown mongo error happens.
+*/
+var ErrGenericDB = errors.New("DB Error")
```

Comment:
Ok, I realized that this is actually unused, so it's not that bad. But then maybe we should remove it completely